### DISCUSSION TITLE
Animation Glitch in Slider

### DISCUSSION THREAD

**Discussion by nikeshpurohit** [Clooos/Bubble-Card#291] [Feature requests]
This only happens on certain devices, the example here, "Living Room Lights" are a Phillips Hue group integrated from Zigbee2mqtt and the "Screen Lights" are from the WLED integration.

When changing brightness the slider bounces back and to the set value:

![Bubble-Card-1](https://github.com/Clooos/Bubble-Card/assets/30273525/9b198208-af13-4ccc-927d-70e22cad158f)

**Comment by Clooos** [comment]
Hi and thanks for your feedback. This is not really a bug, the slider is always synced with the light in real time, so when you slide in it, the second movement of the slider is just the update of the brightness of that light. It only depend on the delay of your light. But I will try to change that.

**Comment by nikeshpurohit** [comment]
Hi, thank you for the reply, that makes sense as those lights have a slow fade when changing brightness. Can I suggest using a transparent layer to separate the target state and the actual state of the slider like the native thermostat interface in home assistant?
![Bubble-Card-2](https://github.com/Clooos/Bubble-Card/assets/30273525/73205ef5-0e28-4707-94fd-85a609f1e1f1)

**Comment by Clooos** [comment]
As the sliders are always synced, the actual state is always known. I not sure that this can be really useful.

**Comment by celestianx** [comment]
Hi,
First and foremost, thank you for your great work. It looks very promising and I can't wait for its future :)
I have the same problem and those lights are either individual or grouped lights from z2m. While I understand what you say, I have not such an issue with standard or mushroom cards so I guess they are doing some under the hood optimizations
https://github.com/piitaya/lovelace-mushroom/blob/main/src/shared/slider.ts

Maybe it is also related but I am seeing this kind of message here at the bottom left [:[](url)](![Overview-–-Home-Assistant](https://github.com/Clooos/Bubble-Card/assets/1412285/93adb7fa-0f10-410e-a35a-0abecb63b7c2))

**Comment by Clooos** [comment]
Hi! I need to take a look at this, thanks for the info!

And that error is only in edit mode, but I guess that I can fix that as well. It's not my priority because I have a lot more to fix before that 😅

But thanks again for the feedback!

**Comment by celestianx** [comment]
Thanks, maybe sometime it's not about having the state entirely synced to the light but rather to the end user action. Maybe the delay is less noticeable that way from and end-user perspective vs. what you have currently implemented!
